iFixit/react-commerce
# Next: Upgrade to v14

NextJS 14 does not have any breaking changes. It should be safe to upgrade!

See:

https://nextjs.org/docs/pages/building-your-application/upgrading/version-14

https://dev.to/mongodb/nextjs-14-no-new-apis-breaking-changes-3nh3

QA
---

* Run `nvm use`
	* It may instruct you to run `nvm install`
	* Then run `nvm use` to ensure you are on node 18.17.0
* Run `npm install -g pnpm@8`
* Run `npm install -g yarn`
* Run `pnpm dev`
* Smoke test site.

Note: I expect no issues as the claim is nothing is breaking, but I'd like to check before merging.

Connects: https://github.com/iFixit/ifixit/issues/50776
